connector/src/main/java/org/geysermc/connector/dump/DumpInfo.java

```diff
@@ -188,6 +194,25 @@ public DumpInfo() {
         }
     }
 
+    @Getter
+    public static class LogsInfo {
+        private String Link;
+
+        public LogsInfo(Boolean addLog) {
+
+            try {
```

Comment:
Check for `addLog` first, then create the try/catch.

Reply:
`addLog` shouldn't even be passed here, since its checked already in `DumpInfo()`. Or don't check in `DumpInfo()` and only check here.

Also, use `boolean` instead of `Boolean`